core/nuxeo-notify-behavior.js

```diff
@@ -0,0 +1,42 @@
+/**
+@license
+(C) Copyright Nuxeo Corp. (http://nuxeo.com/)
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+import '@polymer/polymer/polymer-legacy.js';
+
+let fallback;
+
+export const setFallbackNotificationTarget = (el) => {
+  fallback = el;
+};
+
+/**
+ * @polymerBehavior Nuxeo.NotifyBehavior
+ */
+export const NotifyBehavior = {
+  notify(options) {
```

Comment:
Are there use cases in which it would make sense to not bubble? We could expose `composed` and `bubbles` as defaults to `true` so that they can be changed.
Or not, we can always add those later without any problems.

Reply:
We have no use case for that at the moment.